database/insertData/07_review_assignments.js

```diff
@@ -334,7 +333,6 @@ exports.queries = [
             connectByUsername: { username: "testConsolidator1" }
           }
           status: AVAILABLE_FOR_SELF_ASSIGNMENT
-          templateSectionRestrictions: ["S1", "S2"]
```

Comment:
Same here? (and a few others, I won't list them all)

Reply:
Same reason